src/main/java/frc/robot/RobotContainer.java

```diff
@@ -9,32 +9,46 @@
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
 import frc.robot.commands.DefaultDriveUpdatePose;
+import frc.robot.commands.DropCone;
+import frc.robot.commands.DropCube;
+import frc.robot.commands.FloorIntakeIn;
+import frc.robot.commands.FloorIntakeOut;
+import frc.robot.commands.IntakeCone;
+import frc.robot.commands.IntakeCube;
 import frc.robot.commands.ToggleRotationTarget;
 import frc.robot.subsystems.DrivetrainSubsystem;
+import frc.robot.subsystems.FloorIntake;
+import frc.robot.subsystems.Intake;
 import frc.robot.subsystems.Vision;
 import frc.robot.util.AutoGenerator;
 import frc.robot.util.JoystickModification;
-import frc.robot.commands.IntakeCone;
-import frc.robot.commands.IntakeCube;
-import frc.robot.commands.DropCube;
-import frc.robot.commands.DropCone;
-import frc.robot.subsystems.Intake;
 
 public class RobotContainer {
   private final Vision vision = new Vision();
   private final DrivetrainSubsystem drivetrainSubsystem = new DrivetrainSubsystem();
   private final JoystickModification mod = new JoystickModification();
-  private final Joystick operatorController = new Joystick(Constants.OPERATOR_CONTROLLER);
+
+  /* 
+   * Ground Intake Controller Operation:
+   * -  Push Button once, start the intake motor and set to suck inwards
+   * -  Push the button at anytime during the live state to switch directions
+   * -  Long press the button to shut down the motor
+  */
```

Comment:
This comment is a nice idea but lets wait until all of our controls are mapped in more final state and we could add a nice documentation comment describing our mapping

Reply:
Suggest to remove this comment until a later time after we finalize our control mapping

Reply:
This comment is still here, I like the idea but lets do this at a later time once our controls are finalized